docs/formats/standard.md

```diff
@@ -0,0 +1,40 @@
+# Standard vulnerability format
+
+We provide an high level format that work for all available strategy. It can be activated with the option `useFormat` equal `Standard`.
```

Comment:
```suggestion
We provide a high-level format that works for all available strategies. It can be activated with the option `useFormat` equal `Standard`.
```